scipy/stats/_stats_py.py

```diff
@@ -7069,19 +7073,19 @@ def ttest_ind_from_stats(mean1, std1, nobs1, mean2, std2, nobs2,
 
                           Number of    Sample     Sample
                     Size    ones        Mean     Variance
-        Sample 1    150      30         0.2        0.161073
-        Sample 2    200      45         0.225      0.175251
+        Sample 1    150      30         0.2        0.161073 Sample 2    200 45
+        0.225      0.175251
 
-    The sample mean :math:`\hat{p}` is the proportion of ones in the sample
-    and the variance for a binary observation is estimated by
+    The sample mean :math:`\hat{p}` is the proportion of ones in the sample and
+    the variance for a binary observation is estimated by
```

Comment:
```suggestion
    The sample mean :math:`\hat{p}` is the proportion of ones in the sample
    and the variance for a binary observation is estimated by
```

Reply:
Removed that change in https://github.com/scipy/scipy/commit/8b60b333ccf2df714169909440e9a8273395f5e9.